posthog/models/filters/mixins/funnel.py

```diff
@@ -71,6 +75,42 @@ def microseconds_from_days(days):
         return microseconds * FunnelWindowDaysMixin.milliseconds_from_days(days)
 
 
+class FunnelWindowMixin(BaseParamMixin):
+    @cached_property
+    def funnel_window(self) -> Optional[int]:
+        _amt = int(self._data.get(FUNNEL_WINDOW, "0"))
+        if _amt == 0:
+            return None
+        return _amt
+
+    @cached_property
+    def funnel_window_interval(self) -> Optional[str]:
+        _interval = self._data.get(FUNNEL_WINDOW_INTERVAL, None)
+        return _interval
+
+    @include_dict
+    def funnel_window_to_dict(self):
```

Comment:
I think this can be just a single `to_dict` function with both keys? Only used for the cache, and every other mixin follows this pattern.